prow/github/client.go

```diff
@@ -452,7 +452,11 @@ func (c *Client) requestRetry(method, path, accept string, body interface{}) (*h
 						break
 					}
 				} else if oauthScopes := resp.Header.Get("X-Accepted-OAuth-Scopes"); len(oauthScopes) > 0 {
-					err = fmt.Errorf("is the account using at least one of the following oauth scopes?: %s", oauthScopes)
+					authorizedScopes := resp.Header.Get("X-OAuth-Scopes")
+					if authorizedScopes == "" {
+						authorizedScopes = "no"
```

Comment:
Personally, I don't like setting the `authorizedScopes` to `"no"` here...but I can't figure out how to make the message pretty without it. :thinking: